app/assets/javascripts/thredded.es6

```diff
@@ -1,18 +1,59 @@
+//= require babel/polyfill
 //= require jquery
 //= require jquery_ujs
 //= require_tree ./thredded/vendor
 //= require_tree ./thredded
 
-class Thredded {
-  constructor() {
-    this.topicForm = new ThreddedTopicForm();
-    this.privateTopicForm = new ThreddedPrivateTopicForm();
-    this.postForm = new ThreddedPostForm();
-    this.kbShortcuts = new ThreddedKeyboardShortcuts();
-    this.timestamps = new ThreddedTimeStamps();
-    this.searchBar = new ThreddedSearchBar();
-    this.currentlyOnline = new ThreddedCurrentlyOnline();
+(function() {
```

Comment:
I've seen this approach in more than a few places but never took the time to investigate why. Now that I see it in here I did a little searching and from what I can tell it's to wrap everything in an isolated context so as not to pollute the global js namespace, is that right?

And yes, I do realize my ignorance is showing through via my obvious disdain for javascript. I really have to figure out how to get through that.